tests/infer/mcmc/test_hmc.py

```diff
@@ -298,39 +298,13 @@ def model(data):
 
     true_std = torch.tensor([0.5, 2])
     data = dist.Normal(3, true_std).sample(sample_shape=(torch.Size((2000,))))
-    hmc_kernel = HMC(model, trajectory_length=1, adapt_step_size=True,
+    hmc_kernel = HMC(model, trajectory_length=1, step_size=0.3,
```

Comment:
I suggest to set this to a smaller step_size ((because things are wrong with even very small step_size, I suspect that initial trace is so bad; so we should use a small step_size for mass_adaptation; 0.3 is too large for trajectory_length=1 and bad initial trace), and set `adapt_step_size=False`. You might increase `warmup_steps` to a higher value if needed.

Reply:
I'll just change the rng seed and increase the number of warmup steps. Otherwise we'll need a very small step size without adaptation, and that will take even longer to finish.

Reply:
How about `step_size=0.03, adapt_step_size=False` and `warmup_steps=200`? It works in my machine. I don't see the reason for changing init_step_size and still use adapt_step_size.

Reply:
Ah, by `init_step_size`, I mean the input `step_size`. Sorry for making confusion!

Reply:
Np, I meant step size too! I had changed the random seed without specifying the `step_size`, but this is probably going to be more stable. Relying on fixing the seed isn't a stable strategy specially with the ongoing refactoring of the rng in pytorch.